src/main/resources/templates/fragments.html

```diff
@@ -11,7 +11,7 @@
 <div th:fragment = "pageHeader" class="wrapper">
 
     <div class="header">
-        <a id="login" th:href="@{/login}">Login</a>
+        <a id="login" th:href="@{/}">Login</a>
```

Comment:
Huh... this should be working but is giving me a 404 error and still trying to direct to /login